subsys/net/ip/net_if.c

```diff
@@ -4249,6 +4280,73 @@ int net_if_down(struct net_if *iface)
 	return status;
 }
 
+int net_if_connect(struct net_if *iface)
+{
+	NET_DBG("iface %p connect", iface);
+
+	const struct net_if_conn *connectivity = net_if_get_conn(iface);
```

Comment:
It would be nice if we could follow zephyr style here, so variables introduced in the beginning of the block.
I see this in this function and the other functions too.